flasher.cpp

```diff
@@ -655,28 +631,17 @@ void Flasher::SaveBoardKeyToFile()
     keys_file_.close();
 }
 
-bool Flasher::SendDisconnectCmd()
+bool Flasher::SendMessage(const char *data, qint64 length, int timeout_ms)
```

Comment:
we can also use QByteArray as commands `QByteArray kEraseCmd = "erase";` and `serial_port_.write` will accept that format. When I was writing this I didn't know that.  This will reduce the need for `sizeof` for every `SendMessage`.